src/main/java/org/springframework/data/couchbase/core/ExecutableFindByIdOperation.java

```diff
@@ -17,15 +17,20 @@
 
 import java.util.Collection;
 
+/*
+ * @author Michael Nitschinger
+ * @author Michael Reiche
+ */
+
 public interface ExecutableFindByIdOperation {
 
 	<T> ExecutableFindById<T> findById(Class<T> domainType);
 
 	interface TerminatingFindById<T> {
 
-		T one(String id);
```

Comment:
this is a separate change, right? Why do we need convert from string to object on ids in the auditing change?

Reply:
My test case included a PersonRepository<Person,UUID> which will not work.  one(<T> object) methods fail when @Id is anything other than String, because the code called relies on id being a String.  Without refactoring all the classes and interfaces to be parameterized with <T,ID> instead of just <T>, simply changing String to Object and (eventually) calling id.toString() 

Actually a better solution would be to make the Entity objects parameterized by ID so the repository would be

PersonRepository<Person<UUID>>

Reply:
removed this change.  It's now in a separate branch.